frontend/components/ClickableUrls/ClickableUrls_xss.tests.tsx

```
@@ -0,0 +1,59 @@
+    // Plain text "javascript:" is harmless -- the risk is only when it
+    // appears inside an href attribute. DOMPurify should strip it there.
+    const malicious =
+      // eslint-disable-next-line no-script-url
+      'Click <a href="javascript:alert(\'xss\')">here</a> for details';
+    const { container } = render(<ClickableUrls text={malicious} />);
+    const link = container.querySelector("a");
+    // DOMPurify should either remove the href entirely or strip the
+    // javascript: scheme. Both outcomes are safe.
+    const href = link?.getAttribute("href");
+    if (href !== null && href !== undefined) {
+      expect(href).not.toContain("javascript:");
```

ESLint

[error] Script URL is a form of eval.

(no-script-url)